spec/features/admin_gets_redirected_to_admin_dashboard_after_login_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+RSpec.describe 'Admin', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.